lib/strong_resources/controller/mixin.rb

```diff
@@ -19,14 +18,31 @@ def type
         end
       end
 
-      included do
-        class_attribute :_strong_resources, instance_writer: false
-        self._strong_resources = {}
+      def self.included(klass)
+        klass.class_eval do
+          extend ClassMethods
+
+          class << self
+            attr_accessor :_strong_resources
+            def inherited(subclass)
+              subclass.instance_variable_set(:@_strong_resources, instance_variable_get(:@_strong_resources).dup)
```

Comment:
If this is an accessor, why use `instance_variable_set`?

Will `dup` work, or does this need `deep_dup`?

FWIW I was fine with `class_attribute`, it just needs to be duped on inherited.